### DISCUSSION TITLE
SUA5000RMI5U

### DISCUSSION THREAD

**Discussion by mirceapaun14** [netbox-community/devicetype-library#1918] [General]
Sorry guys, new to github and netbox, i have this device and its missing so i build it up. Don't know how to add it to pull requests :)
The only thing that is wrong is the inlet, i do not know the standard for direct connection on a bracket, so i put a c20 plug.
Cheers 
Mircea

---
manufacturer: APC
model: APC Smart-UPS 5000VA 230V Rackmount/Tower
slug: apc-smart-ups-sua5000rmi5Uu
part_number: SUA5000RMI5U
u_height: 5.0
is_full_depth: true
airflow: front-to-rear
comments: '[Quickspecs](https://www.apc.com/uk/en/product/download-pdf/SUA5000RMI5U?filename=APC_SmartUPS_SUA5000RMI5U.pdf)'
weight: 97.73
weight_unit: kg
console-ports:
- name: Serial
  type: rj-45
power-ports:
- name: Inlet
  type: iec-60320-c20
  maximum_draw: 4000
power-outlets:
- name: Outlet 1
  type: iec-60320-c13
  power_port: Inlet
- name: Outlet 2
  type: iec-60320-c13
  power_port: Inlet
- name: Outlet 3
  type: iec-60320-c13
  power_port: Inlet
- name: Outlet 4
  type: iec-60320-c13
  power_port: Inlet
- name: Outlet 5
  type: iec-60320-c13
  power_port: Inlet
- name: Outlet 6
  type: iec-60320-c13
  power_port: Inlet
- name: Outlet 7
  type: iec-60320-c13
  power_port: Inlet
- name: Outlet 8
  type: iec-60320-c13
  power_port: Inlet
- name: 16A Outlet 1
  type: iec-60320-c19
  power_port: Inlet
- name: 16A Outlet 2
  type: iec-60320-c19
  power_port: Inlet
module-bays:
- name: SmartSlot
  position: Rear

**Comment by kbrown900** [comment]
You can create a fork of this repo, and then make the changes you are wanting to add. After the changes are in you can create a PR draft and see if it can be merged. If you click the "contribute" drop-down in your repo you can open a PR. I recommend doing the pre-commit checks described in the readme to make sure it meets all the quality/standardization requirements of the repo.
